website/blog/2024-01-08-new-ferretdb-v118-support-oplog-functionality.md

```diff
@@ -0,0 +1,89 @@
+---
+slug: new-ferretdb-v-11180-support-oplog-functionality
+title: FerretDB releases v1.18.0!
+authors: [alex]
+description: >
+  We just released FerretDB v1.18.0, which now supports `OpLog` functionality, capped collections, tailable cursors, among other new features.
+image: /img/blog/ferretdb-v1.18.0.jpg
+tags: [release]
+---
+
+![FerretDB releases v.1.18.0](/img/blog/ferretdb-v1.18.0.jpg)
+
+We are starting the new year with a major feature addition!
+The latest version of FerretDB v1.18.0 has just been released with support for basic `OpLog` functionality, along with other exciting features.
+
+`OpLog` functionality has been one of our most requested features, and it would make it possible for developers to build real-time applications using the Meteor framework.
+We made this a priority in the last quarter and are super excited that it's now available.
+We can't wait to see what you build with FerretDB!
+
+In addition to `OpLog` functionality, we've also added support for capped collections and tailable cursors.
+Other new features in the release include support for `createUser`, `dropUser`, `dropAllUsersFromDatabase`, and `UsersInfo`.
+
+FerretDB is on a mission to add MongoDB compatibility to other database backends, including [Postgres](https://www.postgresql.org/) and [SQLite](https://www.sqlite.org/).
+All the new features in this release will help us improve compatibility with more applications，and use cases.
+
+## New features
+
+In this release, we've added **support for capped collection**, along with `max` and `size` parameters.
+
+For example, to create a capped collection `testcollection` with a maximum size of 512MB, run:
+
+```js
+db.createCollection('testcollection', { capped: true, size: 536870912 })
+```
+
+This release also includes **support for tailable cursors**; both `tailable` and `awaitData` parameters are included.
+
+To add a tailable cursor, run:
+
+```js
+db.collection.find(
+  { $query: {}, $orderby: { $natural: 1 } },
```

Comment:
That's not a valid syntax. Where does it come from?

Reply:
That's not supposed to be there. Thanks for sighting this, i should have deleted it before.